docs/nidcpower/class.rst

```diff
@@ -1569,6 +1569,39 @@ wait_for_event
             :type timeout: float in seconds or datetime.timedelta
 
 
+.. role:: c(code)
+    :language: c
+
+.. role:: python(code)
+    :language: python
+
+Repeated Capabilities
+=====================
+
+    Repeated capabilities attributes are used to set the `channel_string` parameter to the
+    underlying driver function call. This can be the actual function based on the :py:class:`Session`
+    method being called, or it can be the appropriate Get/Set Attribute function, such as :c:`niDCPower_SetAttributeViInt32()`.
+
+    Repeated capbilities attributes use the indexing operator :python:`[]` to indicate the repeated capabilities.
+    The parameter can be a string, list, tuple, or slice (range). Each element of those can be a string or
+    an integer. If it is a string, you can indicate a range using the same format as the driver: :python:`'0-2'` or
+    :python:`'0:2'`
+
+    Some repeated capabilities use a prefix before the number and this is optional
+
+channels
+--------
+
+    .. py:attribute:: nidcpower.Session.channels[]
+
+        .. code:: python
+
+            session.channels['0-2'].channel_enabled = True
```

Comment:
Let's use range or a list literal instead which is more pythonic than our convoluted syntax.

Reply:
See above